### ISSUE TITLE
[Server-side][Feature request] Improvements for logs

### ISSUE THREAD

**Issue by yelsukov** [tinode/chat#215]
### Subject of the issue

1) Right now I can see in logs only requests from client, but I can not find in logs any responses for them.

Sometimes It is very hard to debug client side without this logs. Is it possible to add logs for responses from server too?

2) Also I can see in logs private messages of client's  from their requests. It would be better to cut it out or hide from logs. 

### Is this a bug report of a feature request?
- [x ] Feature request

#### Server-side
- [x ] Your own setup:
  * platform Ubuntu 16.04
  * version of tinode server, e.g. `0.15.2\`
  * database backend
  
### Server-side log
```json
{
    "pub": {
        "content": "Hello Neo, here it is the private keys from computer of Zeon: blahblahblah",
        "id": "123398",
        "noecho": true,
        "topic": "usrsU7FqBHy6rQ"
    }
}
```

**Comment by or-else** [comment]
I agree that logging can be improved. I have plans to replace the stock logger with https://github.com/sirupsen/logrus. The only thing that stops me is that it's a lot of work.

> Right now I can see in logs only requests from client app, but I can not find in logs any responses for them.

I can see why it may be desirable, but it can't be always turned on because it would generate too much data. Responses could be large. In order to implement this I need to implement logging levels first, i.e. "error", "warning", "info", "debug", etc. That means I have to switch logging to something like https://github.com/sirupsen/logrus before adding response logging.

BTW, I use client-side logging for debugging responses. 

> Also I can see in logs a private messages of client's, from their requests. It would be better to cut it out or hide from logs.

I suppose it can also be solved with logging levels. At the debug log level the server would log the entire packet including user content, in something like "info" it would only log metadata, such as topic and user IDs.
